packages/babel-parser/src/parse-error.js

```diff
@@ -0,0 +1,211 @@
+// @flow
+
+import { Position } from "./util/location";
+import type { NodeBase } from "./types";
+import {
+  type ParseErrorCode,
+  ParseErrorCodes,
+  type ParseErrorCredentials,
+} from "./parse-error/credentials";
+
+const ArrayIsArray = Array.isArray;
+const {
+  assign: ObjectAssign,
+  defineProperty: ObjectDefineProperty,
+  getPrototypeOf: ObjectGetPrototypeOf,
+  keys: ObjectKeys,
+} = Object;
```

Comment:
Nit: we usually don't protect our code against built-ins modifications (I think runkit even relied on it to implement parser plugins, right?).

Reply:
Got rid of these.